Makefile

```diff
@@ -2,6 +2,7 @@ REGISTRY         ?= docker.io
 ORG              ?= ansibleplaybookbundle
 TAG              ?= latest
```

Comment:
I think if this is something we want then we have to make it more dummy proof. At a minimum I think you must set `TAG` to be something like:

`TAG ?= $(shell git rev-parse short HEAD)`

That will prevent people from accidentally using the makefile to break the "latest" broker.